src/app/features/registration/registration.component.html

```diff
@@ -1 +1,76 @@
-<p>registration works!</p>
+<div>
+  <app-header> </app-header>
+  <div class="registration_container">
+    <form [formGroup]="form" (ngSubmit)="onSubmit()">
+      <legend class="title">Registration</legend>
+      <fieldset class="registration_fieldset">
+        <label class="label" for="email">
+          Name
+          <input
+            class="input"
+            type="text"
+            required
+            formControlName="name"
+            minlength="6" />
+          <div
+            class="alert"
+            *ngIf="name.errors && (name.dirty || name.touched)">
+            <p *ngIf="name.errors?.['required']">Name is required</p>
+            <p *ngIf="name.errors?.['minlength']">
+              Name must be at least 6 characters
```

Comment:
You can make for all repeated strings `shared enums` and use them in two or more forms.

Reply:
added enum